server/startup/init.js

```diff
@@ -1,6 +1,18 @@
+import { Meteor } from "meteor/meteor";
+import SimpleSchema from "simpl-schema";
 import { Reaction, Logger } from "/server/api";
 import LoadData from "./load-data";
 
+// This is needed so that it throws a Meteor.Error as `check()` would do
+// when we call schema.validate() in a Meteor method.
+// https://github.com/aldeed/node-simple-schema/#customize-the-error-that-is-thrown
+SimpleSchema.defineValidationErrorTransform(error => {
+  const ddpError = new Meteor.Error(error.message);
+  ddpError.error = "validation-error";
+  ddpError.details = error.details;
+  return ddpError;
+});
```

Comment:
Not sure if this is the proper file to put this in. Let me know if you want it moved.